src/webcomponents/disease-panel/disease-panel-create.js

```diff
@@ -366,25 +410,32 @@ export default class DiseasePanelCreate extends LitElement {
                             display: {
                                 style: "border-left: 2px solid #0c2f4c; padding-left: 12px; margin-bottom:24px",
                                 collapsedUpdate: true,
-                                view: region => html`
-                                    <div>${region.id} - ${region?.modeOfInheritance || "-"}</div>
+                                view: region => html `
+                                    <div>
+                                        <div>${region?.id}</div>
+                                        <div style="margin: 5px 0">MoI: ${region?.modesOfInheritance?.join(", ") || "-"} (Confidence: ${region?.confidence || "NA"})</div>
+                                        <div class="help-block">
+                                            Location: ${region.coordinates?.[0]?.location || "-"},
+                                            Assembly: ${region.coordinates?.[0]?.assembly || "-"},
+                                        </div>
+                                    </div>
                                 `,
                             },
                             elements: [
                                 {
-                                    title: "Region ID",
+                                    title: "Region ID/Location",
                                     field: "regions[].id",
                                     type: "input-text",
                                     display: {
-                                        placeholder: "Add region...",
+                                        placeholder: "Add a region ID/Location (e.g. 1:134545:2324234) ...",
                                     }
                                 },
                                 {
                                     title: "Mode of Inheritance",
                                     field: "regions[].modesOfInheritance",
                                     type: "select",
                                     multiple: true,
-                                    save: value => value.split(","), // Array when select and multiple
+                                    save: value => value?.split(","), // Array when select and multiple
```

Comment:
I would suggest returning an empty array instead of returning `undefined` in this case.

Reply:
Thanks Josemi, I´ve changed the returned value and the optional chaining of rest of the multiple fields as well.